pages/common/sphinx-build.md

```diff
@@ -0,0 +1,16 @@
+# sphinx-build
+
+> Generates documentation from the files in sourcedir and places it in the outputdir.
+> More information: http://www.sphinx-doc.org/en/master/man/sphinx-build.html.
+
+- Build documentation as "epub" and put it in "builddir":
+
+`sphinx-build -b {{epub}} {{path/to/sourcedir}} {{path/to/builddir}}`
```

Comment:
```suggestion
`sphinx-build -b {{html|epub|text|...}} {{path/to/source_dir}} {{path/to/build_dir}}`
```
Let's use underscores for multi-word token. Also, since there are multiple builders, we can list some of them in the token, as we usually do in cases like this one.